### ISSUE TITLE
Extracting bounding boxes for the remaining SSv2 videos

### ISSUE THREAD

**Issue by rmnthn** [eladb3/ORViT#9]
Hi, for SSv2, the annotations zip folder contains the bounding boxes for ~180k videos. In order to get them for the rest of the videos, could you please provide details about the setup that you used to get the detections? Using pretrained Faster-RCNN (detectron2) directly on the test set yields poor results. Please advise on the network architecture, how to fine-tune, hyperparameters, class labels used, etc. It would be helpful if you could provide the annotations for the remaining test videos as well.
Thanks!

**Comment by eladb3** [comment]
Hi, please follow the details provided in the supplemental of "Something-Else Compositional Action" ([here](https://openaccess.thecvf.com/content_CVPR_2020/supplemental/Materzynska_Something-Else_Compositional_Action_CVPR_2020_supplemental.pdf)).
